tests/unit/dbms/test_dbms.py

```diff
@@ -0,0 +1,37 @@
+from jina.drivers.dbms import _doc_without_embedding
```

Comment:
this unit test should be under `executors` not another level

Reply:
Ah yes, thanks. Done